src/components/EditSections/EditExtendedInfo/ResetPasswordControl/ResetPasswordControl.js

```diff
@@ -42,25 +85,21 @@ class ResetPasswordControl extends React.Component {
           <button
             type="button"
             className={css.resetPasswordButton}
-            onClick={this.openModal}
+            onClick={this.handleClick}
```

Comment:
`handleClick` is not giving the context of what is happening. Could you please rename it?